app/furniture/marketplace/tags/_form.html.erb

```diff
@@ -1,6 +1,10 @@
 <%= form_with(model: tag.location) do |form| %>
-
-<%= render "text_field", attribute: :label, form: form %>
-
-<%= form.submit %>
+  <div class="flex flex-col gap-5">
+    <div>
+      <%= render "text_field", attribute: :label, form: form %>
+      <%= form.label :is_group, "Is this a menu section?" %>
```

Comment:
In general, if there is drift between what we're calling things to a user and how we are naming them in the database, I prefer to start with them aligned.

Reply:
Ah, I missed this doing s/section/group.